hail/python/hail/table.py

```diff
@@ -3255,7 +3251,28 @@ def to_pandas(self, flatten=True):
         :class:`.pandas.DataFrame`
 
         """
-        return Env.spark_backend('to_pandas').to_pandas(self, flatten)
+        table = self.flatten() if flatten == True else self
+        dtypes_struct = table.row.dtype
+        collect_dict = {key: hl.agg.collect(value) for key, value in table.row.items()}
+        table.aggregate(hl.struct(**collect_dict))
```

Comment:
Delete this line, it's not doing anything (you use same expression on next line)